src/include/pbs_ifl.h

```diff
@@ -558,6 +559,18 @@ typedef struct preempt_job_info {
         char	order[PREEMPT_METHOD_HIGH + 1];
 } preempt_job_info;
 
+typedef struct svr_conn {
```

Comment:
Shouldn't this part of pbs_internal.h? Do we really want to expose this internal structure of IFL as public interface?